integration/image/Dockerfile

```diff
@@ -1,16 +1,35 @@
-# Generated on: Wed Jul  8 11:06:05 IST 2020
+# Generated on: Thu 23 Jul 22:19:56 IST 2020
 
 # Generated by: wf-gen from tyk-ci
 
 FROM debian:buster-slim
 
+RUN apt-get update \
+ && apt-get upgrade -y \
+ && apt-get install -y --no-install-recommends \
+        wget jq curl ca-certificates apt-transport-https gnupg unzip
+
+RUN apt-get install -y --no-install-recommends \
+        build-essential \
+        python3-setuptools \
+        libpython3.7 \
```

Comment:
If Python is for the plugins here, it might also need `python3.7-dev` package — otherwise libpython won't be correctly detected by the coprocess loader (see https://github.com/TykTechnologies/tyk-gateway-docker/blob/f35efea072e3e9a613124537bde9b82c77969d82/Dockerfile#L18 and https://github.com/TykTechnologies/tyk-gateway-docker/issues/35 for context).